django/contrib/auth/management/commands/changepassword.py

```diff
@@ -13,6 +13,7 @@ class Command(BaseCommand):
     help = "Change a user's password for django.contrib.auth."
     requires_migrations_checks = True
     requires_system_checks = False
+    private_options = ('stderr', 'stdout')
```

Comment:
Options like `no_color`, `stdout`, `stderr`, skip_checks` should be accepted for all commands, https://github.com/django/django/blob/545a88fc162ab02f66812284af24a248deafbf5e/django/core/management/base.py#L305-L340